inst/utils/golem_utils_ui.R

```diff
@@ -294,6 +294,80 @@ col_1 <- function(...){
   column(1, ...)
 }
 
+
+
+#' Make the current tag behave like an action button
+#' 
+#' Only works with compatible tags like button or links
+#'
+#' @param tag Any compatible tag.
+#' @param inputId Unique id. This will host the input value to be used
+#' on the server side.
+#'
+#' @return The modified tag with an extra id and the action button class.
+#' @noRd
+#'
+#' @examples
+#' if (interactive()) {
+#'  library(shiny)
+#'  
+#'  link <- a(href = "#", "My super link", style = "color: lightblue;") 
+#'  
+#'  ui <- fluidPage(
+#'   make_action_button(link, inputId = "mylink")
+#'  )
+#'  
+#'  server <- function(input, output, session) {
+#'    observeEvent(input$mylink, {
+#'     showNotification("Pouic!")
+#'    })
+#'  }
+#'  
+#'  shinyApp(ui, server)
+#'  
+#' }
+make_action_button <- function(tag, inputId = NULL) {
+  # some obvious checks
+  if (!inherits(tag, "shiny.tag")) stop("Must provide a shiny tag.")
+  if (!is.null(tag$attribs$class)) {
+    if (grep("action-button", tag$attribs$class)) {
+      stop("tag is already an action button")
+    }
+  }
+  if (is.null(inputId) && is.null(tag$attribs$id)) {
+    stop("tag does not have any id. Please use inputId to be able to
+           access it on the server side.")
+  }
+  
+  # handle id
+  if (!is.null(inputId)) {
+    if (!is.null(tag$attribs$id)) {
+      cat_red_bullet(
+        paste(
+          "tag already has an id. Please use input$", tag$attribs$id, "to access it from the server side. inputId will be ignored.")
+      )
+    } else {
+      tag$attribs$id <- inputId
+      cat_green_tick("Adding id attribute ...")
```

Comment:
I think these confirmation might not be useful and might add unnecessary console output when launching the app. 

I mean, if I have correctly configured them, I would expect them to be silent